README.md

```diff
@@ -58,9 +58,64 @@ Note for Ubuntu: Please note that the uncomplicated firewall (ufw) is known to c
 
 ## Deployment
 
+### Site name
+
+You will need to choose a short name for your site. This is not a URL, just a simple identifying string. For the examples below, we will use "your-site-name", but you should obviously choose something that is meaningful to you and which is unique.
+
+Site names should adhere to the following conventions:
+
+- They should be lower-case.
+- They should generally be named after the city where your site is based, e.g. ```karlsruhe```.
+- If you have a multi-part name, please use a hypen ("-") as separator, e.g. ```le-havre```.
+- If your site is for testing purposes, rather than production, please append "-test", e.g. ```zaragoza-test```.
+- If you are a developer and you are making changes to the Bridgehead, please use your name and append "-dev", e.g. ```joe-doe-dev```.
+
+### Projects
+
+The following "projects" are known to the Bridgehead installation:
+
+- bbmri
+- ccp
+
+Use "bbmri" if you are in the BBMRI-ERIC European biobank network or the GBA (German Biobank Alliance) network.
+
+Use "ccp" if you are in the DKTK network, the C4 network or the nNGM network.
+
+### GitLab repository
+
+In order to be able to install, you will need to have your own repository in GitLab for your site's configuration settings. This allows automated updates of the Bridgehead software.
+
+To request a new repository, please send an email to one of the following:
+
+- For the bbmri project: bridgehead@helpdesk.bbmri-eric.eu.
+- For the ccp project: support-ccp@dkfz-heidelberg.de
+
+Mention which project you belong to, i.e. "bbmri" or "ccp", plus your chosen site name.
+
+We will set the repository up for you. We will then send you:
+
+- A Repository Short Name (RSN). Beware: this is distinct from your site name.
+- The repository's URL.
+- A token to access the repository.
+
+Before installation, you must set up your site's configuration in GitLab.
+
+To do this, visit the configuration repository's URL and click on the configuration file. Depending on your project, this will be called either ```bbmri.conf```or ```ccp.conf```. Use the blue button to edit it. You will need to change, as a minimum, the following variables:
+
+- SITE_NAME
+- SITE_ID
+- OPERATOR_FIRST_NAME
+- OPERATOR_LAST_NAME
+- OPERATOR_EMAIL
+- OPERATOR_PHONE
+
+SITE_NAME and SITE_ID can be set to the chosen name for your site, e.g. "your-site-name". OPERATOR_* should be set to values appropriate for the administrator of your site.
+
+Once you have made your changes, these will need to be reviewed by members of our team before you can proceed with the installation.
+
```

Comment:
I agree with @TKussel that this should be done on our side. However, it might be smart to do the following: **As part of the installation**, the BK admin receives a message like "Just checked out your config repository from the central git repository at https://... Please click on this link to review your site-specific configuration.

```suggestion
During the installation, your Bridgehead will download your site's configuration from GitLab. You will receive a weblink to review these settings and make changes as needed. To do this, visit the URL and click on the configuration file (```*.conf```, depending on your network). Use the blue button to edit it. You will need to check, as a minimum, the following variables:

- SITE_NAME
- SITE_ID
- OPERATOR_FIRST_NAME
- OPERATOR_LAST_NAME
- OPERATOR_EMAIL
- OPERATOR_PHONE

SITE_NAME and SITE_ID should be set to the chosen name for your site. OPERATOR_* should be set to values appropriate for the administrator of your site (see examples in the file).

Once you have made your changes, these will need to be reviewed by members of our team as part of a git pull request. Once accepted, the Bridgehead will automatically re-download these settings as part of its auto-update.

```

Reply:
Do we want to deal with access rights to gitlab? I don't think they can view the repo without an account, access token is not enough. Would just include the info in the email. No need to bother them with the details about setting the site Id as it should be done on our side. They can review it when they clone the repo